api/clientset/v1alpha1/srlinux.go

```diff
@@ -108,14 +104,14 @@ type srlinuxClient struct {
 // List gets a list of SRLinux resources.
 func (s *srlinuxClient) List(
 	ctx context.Context,
-	opts *metav1.ListOptions,
+	opts metav1.ListOptions,
 ) (*typesv1alpha1.SrlinuxList, error) {
```

Comment:
@marcushines can you share your expert opinion on that func signature and the need for ListOptions to be copied?

The reason I used a pointer here is based on a purely noob approach - a code analyzer told me that the struct is "sizeable", so why not using a pointer?

Reply:
I see, this is a common interface pattern used in k8s, probably for a good reason
https://pkg.go.dev/k8s.io/client-go@v0.24.3/kubernetes/typed/core/v1#PodInterface